src/explorer/Main.hs

```diff
@@ -43,19 +46,28 @@ import           Pos.Util.UserSecret        (usVss)
 import           Pos.Util.Util              (powerLift)
 import           Pos.WorkMode               (RealMode, WorkMode)
 
+import           Pos.Explorer               (ExplorerBListener,
+                                             runExplorerBListener)
 import           Pos.Explorer.Socket        (NotifierSettings (..))
 import           Pos.Explorer.Web           (explorerPlugin, notifierPlugin)
 
 import           ExplorerOptions            (Args (..), getExplorerOptions)
 import           Params                     (getBaseParams, getKademliaParams,
                                              getNodeParams, gtSscParams)
 
+
+type ExplorerProd = ExplorerBListener (RealMode SscGodTossing)
+
+-- | Lift monadic computation to 'ExplorerBListener'.
+liftBListenerRedirects :: m a -> ExplorerBListener m a
+liftBListenerRedirects = coerce
```

Comment:
@akegalj `coerce` changes value of type `a` to the same value of type `b` if they are same internally.
For example, `ExplorerBListener` is tagged `IdentityT` (which is a newtype over `f a`), so types are same internally.
`lift` could also be used in this particular case, because `IdentityT` is a monad transformer.
However, in general `coerce` is not a synonym for `lift`.